mm2src/mm2_bin_lib/Cargo.toml

```diff
@@ -22,14 +22,14 @@ track-ctx-pointer = ["mm2_main/track-ctx-pointer"]
 zhtlc-native-tests = ["mm2_main/zhtlc-native-tests"]
 
 [[bin]]
-name = "mm2"
+name = "kdf"
 path = "src/mm2_bin.rs"
 test = false
 doctest = false
 bench = false
```

Comment:
> > Providing a copy with mm2 name next to kdf binary would be fine for a while (~3 months)
> 
> This is for the release artifacts, right? we don't need to do this in CI, or did you mean that?

For anything (release, CI builds, containers, etc). If you duplicate this line with old name, cargo should produce 2 binaries (mm2 and kdf). I think we don't spend too much time on building the final binary output, so build time should not increase too much.

Reply:
> cargo should produce 2 binaries (mm2 and kdf)

It does that, but I get a warning

```bash
warning: /atomicDEX-API/mm2src/mm2_bin_lib/Cargo.toml: file found to be present in multiple build targets: /atomicDEX-API/mm2src/mm2_bin_lib/src/mm2_bin.rs
```
I will leave it. I can't duplicate the [lib] block so I will leave it as `kdflib` only. Will work on CI and other now.